types/three/three-core.d.ts

```diff
@@ -1447,6 +1447,15 @@ export class Geometry extends EventDispatcher {
     dispatchEvent(event: { type: string; [attachment: string]: any; }): void;
 }
 
+/**
+ * @see <a href="https://github.com/mrdoob/three.js/blob/master/examples/js/BufferGeometryUtils.js">examples/js/BufferGeometryUtils.js</a>
+ */
+export namespace BufferGeometryUtils {
+    export function mergeBufferGeometries(geometries: any[]): BufferGeometry;
+    export function computeTangents(geometry: BufferGeometry): null;
+    export function mergeBufferAttributes(attributes: any[]): BufferAttribute;
```

Comment:
Is this an array of `any`?

Reply:
Again, should be an array of `BufferAttribute`s

```typescript
export function mergeBufferAttributes(attributes: BufferAttribute[]): BufferAttribute;
```

Reply:
Done.